internal/command/postgres/update.go

```diff
@@ -192,6 +194,14 @@ func runUpdate(ctx context.Context) error {
 		}
 	}
 
+	if !flag.GetBool(ctx, "detach") {
```

Comment:
Instead of running health checks at the end of the update process, we should look to monitor the health of the individual machine after each update.  Ideally the cluster should be in a healthy state before any new operation is executed. 

E.G.
Update Machine 1  ( Wait for passing health checks )
Update Machine 2 ( Wait for passing health checks )
Step down ...          ( Wait for passing health checks )